Dockerfile

```diff
@@ -18,8 +18,7 @@ RUN yarn build
 FROM python:3.8.17-slim-bullseye
 LABEL maintainer="Hypothes.is Project and contributors"
 
-# Install nginx & supervisor
-RUN apt-get update && apt-get install --yes nginx nginx-extras gettext-base supervisor \
+RUN apt-get update && apt-get install --yes nginx nginx-extras gettext-base supervisor libpq-dev \
```

Comment:
I _think_ `libpq-dev` (which contains the `pg_config` command) needs to be left in the container, it's not only needed for the `pip install` command.